airbyte-cdk/python/airbyte_cdk/sources/declarative/models/declarative_component_schema.py

```diff
@@ -1207,6 +1221,9 @@ class Config:
         description='A list of transformations to be applied to each output record.',
         title='Transformations',
     )
+    state_migrations: Optional[
+        List[Union[LegacyToPerPartitionStateMigration, CustomStateMigration]]
+    ] = []
```

Comment:
It's better to use `None` here, or the `Field(default_factory=[])`. It's also missing the `description` and example parts, should we add it for consistency between the listed components. See the `transformations` above, as an example.

Reply:
@girarda is this file out of sync or have you not run the code generation since the last edit? Based on the `.yaml`, I would've thought that `description` at a minimum would get populated here as per @bazarnov comment as well

Reply:
yup. good catch! Re-generated the file